optaplanner-docs/src/main/asciidoc/QuickStart/VaccinationScheduler/VaccinationSchedulerQuickStart-chapter.adoc

```diff
@@ -0,0 +1,275 @@
+** Ideal date: Administer the vaccine on or as close to the specified date as possible. For example, if a person receives a second dose, administer it on the ideal date for the specific vaccine, for example 28 days after the first dose. This constraint is softer than the distance constraint to avoid sending people halfway across the country just to be one day closer to their ideal date.
+** Priority rating: Schedule people with a higher priority rating earlier in the planning window. This constraint is softer than the distance constraint to avoid sending people halfway across the country. This constraint is also softer than the ideal date constraint because the second dose is prioritized over priority rating.
+
+Hard constraints are weighted against other hard constraints. Soft constraints are weighted against other soft constraints. However, hard constraints always take precedence over medium and soft constraints. If a hard constraint is broken, then the plan is not feasible. But if no hard constraints are broken then soft and medium constraints are considered in order to determine priority.
+Because there are often more people than available appointment slots, you must prioritize. Second dose appointments are always assigned first to avoid creating a backlog that would overwhelm your system later. After that, people are assigned based on their priority rating. Everyone starts with a priority rating that is their age. Doing this prioritizes older people over younger people. After that, people that are in specific priority groups receive, for example, a few hundred extra points. This  varies based on the priority of their group. For example, nurses might receive an extra 1000 points. This way, older nurses are prioritized over younger nurses and young nurses are prioritized over people who are not nurses. The following table illustrates this concept:
+
+.Priority rating table
+[cols="2,2,3", options="header"]
+|===
+|Age
+|Job
+|Priority rating
+
+|60
+|nurse
+|1060
+
+|33
+|nurse
+|1033
+
+|71
+|retired
+|71
+
+|52
+|office worker
+|52
+|===
+
+
+[id="optaplanner-solver-con_{context}"]
+== The OptaPlanner solver
+
+At the core of OptaPlanner is the solver, the engine that takes the problem data set and overlays the planning constraints and configurations. The problem data set includes all of the information about the people, the vaccines, and the vaccination centers. The solver works through the various combinations of data and eventually determines an optimized appointment schedule with people assigned to vaccination appointments at a specific center.  The following illustration shows a schedule that the solver created:
+
+ifdef::OPTAPLANNER-COMM[]
+image::QuickStart/VaccinationScheduler/vaccinationSchedulingValueProposal.png[]
+endif::OPTAPLANNER-COMM[]
+ifdef::OPTAPLANNER-ENT[]
+image::optimizer/vaccinationSchedulingValueProposal.png[]
+endif::OPTAPLANNER-ENT[]
```

Comment:
If we need to switch paths between community and product for every image, this is going to be hell maintenance wise. We should get rid of this complexity.